libs/net/entities/Models/ContentVersion.cs

```diff
@@ -56,6 +56,10 @@ public class ContentVersion
     /// </summary>
     public bool IsPrivate { get; set; } = false;
 
+    /// <summary>
+    /// get/set - Tone for the content
+    /// </summary>
+    public int? Tone {get;set;}
```

Comment:
Are you even using this property?

Reply:
No, this is from the previous implementation. Thank you!